src/test/ui/suggestions/try-operator-on-main.stderr

```diff
@@ -11,12 +11,34 @@ error[E0277]: the `?` operator can only be used in a function that returns `Resu
    = note: required by `std::ops::Try::from_error`
 
 error[E0277]: the trait bound `(): std::ops::Try` is not satisfied
-  --> $DIR/try-operator-on-main.rs:18:5
+  --> $DIR/try-operator-on-main.rs:20:5
    |
-18 |     try_trait_generic::<()>();
+20 |     ()?;
+   |     ---
+   |     |
+   |     the trait `std::ops::Try` is not implemented for `()`
+   |     in this macro invocation
+   |
+   = note: required by `std::ops::Try::into_result`
+
+error[E0277]: the trait bound `(): std::ops::Try` is not satisfied
+  --> $DIR/try-operator-on-main.rs:23:5
+   |
+23 |     try_trait_generic::<()>();
    |     ^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::ops::Try` is not implemented for `()`
    |
    = note: required by `try_trait_generic`
 
-error: aborting due to 2 previous errors
+error[E0277]: the trait bound `(): std::ops::Try` is not satisfied
```

Comment:
It would be neat to also add a custom error message for this case along the lines of 

    the `?` operator can only be applied to elements that implement the trait `std::ops::Try`

Reply:
I think now that we have this feature one of the docs guys should use it (or variants of it) in all the traits.